pyipmi/interfaces/ipmbdev.py

```diff
@@ -0,0 +1,176 @@
+import os
+import select
+import time
+from array import array
+
+from ..msgs import create_message, encode_message, decode_message
+from ..errors import IpmiTimeoutError
+from ..logger import log
+from ..interfaces.ipmb import IpmbHeaderReq, checksum, rx_filter, encode_ipmb_msg
+
+
+class IPMBDev(object):
+    """This interface uses ipmb-dev-int linux driver."""
+
+    NAME = 'ipmbdev'
+
+    def __init__(self, slave_address=0x20, port=0):
+        # TODO: slave address is currently not defined here
+        self.slave_address = slave_address
+        self.timeout = 0.25
+        self.max_retries = 3
+        self.next_sequence_number = 0
+
+        self._dev = os.open('/dev/ipmb-{}'.format(port), os.O_RDWR)
+
+    def raw_write(self, address, data):
+        # self._dev.i2c_master_write(address, data)
+        pass
```

Comment:
?

Reply:
removed - raw_write() is present in aardvark but is not used anywhere at all